### ISSUE TITLE
gettext workflow description

### ISSUE THREAD

**Issue by AlexMost** [c-3po-org/c-3po#6]

**Comment by erickwilder** [comment]
@AlexMost would you elaborate more on this issue? What do you expect to be added to the docs?

**Comment by AlexMost** [comment]
The idea was to create a doc for someone who doesn't know anything about gettext. [Official docs](https://www.gnu.org/software/gettext/manual/gettext.html) are too verbose to my opinion.

**Comment by erickwilder** [comment]
Agreed. We could add a section "Gettext Primer" or just augment the "Getting Started"section in the docs. 

What do you think?

**Comment by AlexMost** [comment]
Yes, something like "What is gettext" or "Gettext primer" would be useful. As for "Getting started" - there is already [Quick Start](https://ttag.js.org/docs/quickstart.html) doc.
